src/View/Components/_displayCrimeScene.html.twig

```diff
@@ -1 +1,25 @@
-<h1>auteur {{crimeScene.name}}</h1>
+
+<section>
+        <h1> {{crimeScene.title}} </h1>
+    <div>
+        <h1> Auteur: {{crimeScene.name}} </h1>
+    </div>
+
+    <div>
+        <h1> Date : {{crimeScene.date}} </h1>
+    </div>
+    
+    <div>
+        <h1>Adresse : {{crimeScene.adress}} </h1>
+    </div>
+
+    <div>
+        <h1>Victime : {{crimeScene.victim}} </h1>
+    </div>
+    
+    <div>
+        <h1> Description du meurtre : {{crimeScene.description}} </h1>
+    </div>
```

Comment:
you should have 1 h1 only in your section: keep it for the title. use more adapted tags for other fields

Reply:
replace h1 for h2 except for the main title.

Reply:
Same for h2 only one in the section, use other tags, you have only one title in your page
your fields are not titles ;)